### ISSUE TITLE
Kodi-64bit V19 - External Player: VLC-64bit v3.0.12 works, PowerDVD-64bit v19 does not works

### ISSUE THREAD

**Issue by liberopadano** [xbmc/xbmc#19333]
### Describe the bug
Here is a clear and concise description of what the problem is:

The external player VLC-64bit v3.0.12 starts up, plays the movie, returns control to Kodi
The  external player PowerDvd-64bit v19 does not start and kodi hides and immediately becomes visible again

In the previous installation (Kodi-64bit v18 and PowerDvd-32bit v16) the same playcorefactory always worked correctly (obviously with other paths)

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

That the call to PowerDvd-64bit works correctly

### To Reproduce

1.  install kodi-64bit v19.0
2.  install PowerDvd-64bit v19
3.  customize playercorefactoring as mine (see below)
4.  play movie with external player Powerdvd-64bit v19
5.  PowerDvd do not starts

### Debuglog

The debuglog can be found here:

https://paste.kodi.tv/anetadavaj.kodi

If I copy the line #668 of the debug into a batch file it's works!

cmd: "C:\Program Files\CyberLink\PowerDVD19\PowerDVD.exe" "G:\Film\Soylent Green (1973)\2D 1080p\Soylent Green (1973).[2D 1080p].mkv"

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

This is playercorefactory used:

<?xml version="1.0" encoding="UTF-8"?>
<playercorefactory>

  	<players>
		<player name="VLC" type="ExternalPlayer" audio="false" video="true">
       			<filename>C:\Program Files\VideoLAN\VLC\vlc.exe</filename>
			<args>"{1}"</args>
       			<hidexbmc>true</hidexbmc>
		</player>
		<player name="PowerDVD" type="ExternalPlayer" audio="false" video="true">
       			<filename>C:\Program Files\CyberLink\PowerDVD19\PowerDVD.exe</filename>
			<args>"{1}"</args>
       			<hidexbmc>true</hidexbmc>
		</player>
	</players>
  	<rules action="overwrite">
		
		<rule name="defualt-video" audio="false" video="true" player="dvdplayer"/>
  	</rules>

</playercorefactory>

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] OSX
 - [x ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Windows 10 64-bit 
 - Kodi version: 19.0.0

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by DaVukovic** [comment]
I highly guess it's because of: 

CExternalPlayer::Process: Kodi cannot stay hidden for a launcher process

and 

<hidexbmc>true</hidexbmc>

and as you said, it works if you copy the command to a batch file, the default config is:

![image](https://user-images.githubusercontent.com/7235787/110072781-9c814180-7d7e-11eb-80d9-fd674e8c6519.png)

So "hidexbmc" is set to "true" due to your config and "hideconsole" is set to "false" due to the default config settings, could you please test what happens if you remove the "hidexbmc"-option from your config (and probably start Kodi in windowed mode).

**Comment by liberopadano** [comment]
Good news!

First: an update on your suggestion.

I modified my playercorefactory like you said.
Kodi starts without window.
I choose external player PowerDVD.
It do not starts.
Kodi show me a form (translate from italian)
title: external player active	
text: hit 'ok' at the end of playback
command: 'ok'
I hit 'ok'.
Kodi sets movie with 'seen'.

I wrote "If I copy debug line # 668 into a batch file, it works!"

I meant:

I copied the line to a batch file.

I launched the file from the Windows console (so outside of Kodi).
It works.
Conclusion: The command line produced by Kodi is correct.

I launched the batch file via playercorefactory.
Does not work.
Conclusion:
Perhaps the Kodi environment and the PowerDvd environment are not compatible

Now: the solution.

I tried with all exe in PowerDvd folder: with PDVDLP.exe works!
What has changed compared to 'Kodi v18'/'PowerDvd v16' (PowerDVD.exe for the launch of the external player) I don't know, however it works and for me it is fine.

Thank you so much for the time you gave me!
